src/index.ts

```diff
@@ -142,14 +148,18 @@ function parsePr(prList: any, items: any): any[] {
 async function getCommits() {
   try {
     //TODO: add a url composer to handle diffference repo and different time span.
+    const startTime = new Date(Date.now() - 24 * 60 * 60 * 1000).toISOString();
+    const endTime = new Date().toISOString();
```

Comment:
Do we want to consider containing all commits of yesterday instead of 24 hrs ahead of now?
and whats the timezone here? is it standard UTC time of something else?

Reply:
pengpengpeng